libraries/rush-lib/src/cli/parsing/SelectionParameterSet.ts

```diff
@@ -237,14 +240,23 @@ export class SelectionParameterSet {
       })
     );
 
+    const subspaceProjects: Iterable<RushConfigurationProject> = subspaceName
+      ? await this._selectorParserByScope.get('subspace')!.evaluateSelectorAsync({
+          unscopedSelector: subspaceName,
+          terminal,
+          parameterName: 'subspace'
+        })
+      : [];
```

Comment:
@dmichon-msft is there a perf concern if this sort of expression explodes into a set with 1000+ items?  Does the `Selection` system have some efficient way to handle such sets? Or is it fast in practice?

Reply:
If it becomes a problem we can replace the project selection with a bitvector, which makes set theoretic operations much faster.

`Selection.union` is `O(elements)`
`Selection.expandAllDependencies` is `O(projects * directDependencies)`
`Selection.expandAllConsumers` is `O(projects * directConsumers)`

So performance is bounded at `O(projects^2)`. We can profile, but likely not a significant issue compared to some of the places in Webpack that were doing complex set arithmetic.